resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/kubernetes/integrationtest/KubernetesSuite.scala

```diff
@@ -95,6 +97,22 @@ private[spark] class KubernetesSuite extends SparkFunSuite with BeforeAndAfter {
       "changeit")
     keyStoreFile = keyStore
     trustStoreFile = trustStore
+    minikubeKubernetesClient.services().inNamespace("kube-system").createOrReplaceWithNew()
```

Comment:
Hm, is there a way for a service to proxy to pods that are not in its namespace?